README.md

```diff
@@ -179,11 +177,10 @@ When the installation process completes, a number of new VMs will be available o
 
 ### Production and Staging Environments
 
-As of February 2020, the production and staging environments are experimental. If you are interested in becoming involved in development, we recommend you follow the instructions for setting up the [development environment](#development-environment) instead.
+As of February 2020, the production and staging environments are experimental: they should be used on dedicated machines only (i.e. do not install the staging or production environments on your primary workstation). If you are interested in becoming involved in development, we recommend you follow the instructions for setting up the [development environment](#development-environment) instead.
```

Comment:
In this section we might want to discourage usage of staging setup. Is the prod setup still considered experimental ? At this stage, is it not at least in beta/limited release?